app/helpers/application_helper.rb

```diff
@@ -7,6 +7,17 @@ module ApplicationHelper
   include StiRoutingHelper
   include TextualSummaryHelper
 
+  def icon_text(i, text = "", opts = {})
```

Comment:
Uh, I think hardcoding `glyphicon glyphicon-` may be problematic, since we're using 4 different icon fonts - fontawesome, glyphicon, pficon and product (whatever that one is).

That said, there is a `listicon_glyphicon_tag` in [application_helper](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper.rb#L1339), which tries to do a similar thing but the bulk of it's code is a big case of cases.

There's also `toolbar_image` in [toolbar_helper](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/toolbar_helper.rb#L105) which is probably too specific.

Aand finally `active_icon` / `inactive_icon` in [configuraion_view_helper](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/configuration_helper/configuration_view_helper.rb#L22), equally as specific.

Some I'm OK with such function, but I'd prefer more flexibility in the class name, so that non-glyphicon icons can be used as well.